meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -25,27 +25,42 @@ inherit go
 
 inherit systemd
 
-SYSTEMD_SERVICE_${PN} = "mender.service"
-FILES_${PN} += "${systemd_unitdir}/system/mender.service \
+SYSTEMD_SERVICE_${PN} = "mender-client.service"
+FILES_${PN} += "${systemd_unitdir}/system/mender-client.service \
                 ${sysconfdir}/mender.conf \
                 /data/mender/device_type \
                 /data/mender/mender.conf \
                 ${sysconfdir}/udev/mount.blacklist.d/mender \
+                /usr/share/mender/identity \
```

Comment:
Reviewers Note: I have no idea why these were not needed previously...

Reply:
Or whether or not they are all needed now...

Reply:
It has become stricter in zeus. I had to add several of these as well.

Reply:
Btw, I think you can just add `/usr/share/mender/*`.

Reply:
Should we not keep them explicit?

Reply:
It might be better for visibility and catching errors, and there aren't a huge number, so sure, it makes sense to keep them explicit.